src/drivers/generic/dummy-dma.c

```diff
@@ -458,7 +458,7 @@ static int dummy_dma_probe(struct dma *dma)
 		dma->chan[i].dma = dma;
 		dma->chan[i].index = i;
 		dma->chan[i].status = COMP_STATE_INIT;
-		dma->chan[i].private = &chanp[i];
+		dma->chan[i].priv_data = &chanp[i];
```

Comment:
Can you replace it with `dma_chan_set_data(&dma->chan[i], &chanp[i]);`? In a separate patch. Thank you for showing that I am accessing this directly which isn't the best thing to do.

Reply:
Thanks.